webui/ocw/lib/vault.py

```diff
@@ -75,7 +81,12 @@ def getCredentials(self):
 
     def getData(self, name=None):
         if self.auth_json is None:
-            self.auth_json = self.getCredentials()
+            if self.use_file_cache:
+                self.auth_json = self.loadFileCache()
+            if self.auth_json is None:
+                self.auth_json = self.getCredentials()
+                if self.use_file_cache:
```

Comment:
something wrong here , before I will start explain you what tricky situations we could have here can you please double check that you haven't mess up with indentation here ?

Reply:
don't know what you mean. The code changed slightly now here, but this `if` still exists.